demisto_sdk/commands/common/update_id_set.py

```diff
@@ -799,6 +799,48 @@ def get_widget_data(path):
     return {id_: data}
 
 
+def get_dashboard_data(path):
+    data = OrderedDict()
+    json_data = get_json(path)
+    layouts = json_data.get('layout', {})
+    return parse_dashboard_report_data(layouts, data, json_data, path)
+
+
+def get_report_data(path):
+    data = OrderedDict()
+    json_data = get_json(path)
+    layouts = json_data.get('dashboard', {}).get('layout')
+    return parse_dashboard_report_data(layouts, data, json_data, path)
+
+
+def parse_dashboard_report_data(layouts, data, json_data, path):
+    id_ = json_data.get('id')
+    name = json_data.get('name', '')
+    fromversion = json_data.get('fromVersion')
+    toversion = json_data.get('toVersion')
+    pack = get_pack_name(path)
+    report_scripts = []
```

Comment:
It might be a good idea to use a `set` and not a `list`

Reply:
This is a problem, cause there are functions that are expecting this to be a list (like done in script and etc).

Reply:
You can use a set at the beginning and then changing it back to list at the end?